docker/Dockerfile

```diff
@@ -0,0 +1,68 @@
+############################################
+# Dockerfile to run the PCIC data portal   #
+# Based on Ubuntu 16.04                    #
+############################################
+
+FROM ubuntu:16.04
+MAINTAINER Carl Masri <cmasri@uvic.ca>
+
+# Get the system level dependencies
+RUN apt-get update && apt-get install -y \
```

Comment:
It might be good to separate out the "apt-get update" and the install into two separate RUN commands. That way we get the cached image after an apt-get update, and during development, any time that we need to change the system depends, we don't have to run the apt-get update.

This advice can be generalized... in your Dockerfile always do the more expensive operations as early as possible to get the best caching benefits. Obviously order is important, but in places where it doesn't matter, use cost as a tie breaker as for which operation goes first.